src/components/NotificationsModal/NotificationUserListTitle.tsx

```diff
@@ -0,0 +1,55 @@
+import { NotificationUserListTitleFragment$key } from '__generated__/NotificationUserListTitleFragment.graphql';
```

Comment:
this may seem a bit silly, but we don't know what title we want to render on the notification user list modal until the data loads, so we put it in its own component so we can wrap it with a suspense boundary and not block the whole modal